test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.2</TargetFramework>
+    <AssemblyName>Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test</AssemblyName>
+    <OutputType>Exe</OutputType>
```

Comment:
If you add the import to common.props, this also can be removed. In the next update, this property won't be necessary.